utilities/transactions/write_prepared_txn.cc

```diff
@@ -236,14 +236,6 @@ Status WritePreparedTxn::CommitInternal() {
                           NO_REF_LOG, DISABLE_MEMTABLE, &seq_used, ONE_BATCH,
                           &update_commit_map_with_aux_batch);
   assert(!s.ok() || seq_used != kMaxSequenceNumber);
-  if (UNLIKELY(!db_impl_->immutable_db_options().two_write_queues)) {
```

Comment:
This cannot be true.
If `!two_write_queues_` is true, then in line 177, we would have `do_one_write` being true. Therefore, the `if` block starting from line 203 will be entered, causing the function to return from there.